### DISCUSSION TITLE
Add sentence movement to meow

### DISCUSSION THREAD

**Discussion by kaptem** [meow-edit/meow#476] [Ideas]
I think adding meow-next-sentence and meow-back-sentence to meow-command.el would be a good idea and not too hard to implement.

**Comment by DogLooksGood** [comment]
I recommend something like:
```
(add-to-list 'meow-char-thing-table
             '(?{ . sentence))
(add-to-list 'meow-char-thing-table
             '(?} . sentence))
(define-key meow-normal-state-keymap (kbd "{") 'backward-sentence)
(define-key meow-normal-state-keymap (kbd "}") 'forward-sentence)
```

Then do `thing-command } } }` or `thing-command { { {`.

Check this issue as well(for paragraph) #127
